self-hosted-login/okta-aspnet-mvc-example/Content/Site.css

```diff
@@ -20,5 +20,5 @@
 input,
 select,
 textarea {
-    max-width: 350px;
+    max-width: 280px;
```

Comment:
did you mean to change this? I wonder if this breaks some CSS settings.

Reply:
I think I did not change this manually, but I recreated most of the projects in this repo. 
This was changed most likely because of updated project template.